source/quick-start.txt

```diff
@@ -94,7 +94,9 @@ Query Your MongoDB Cluster from Your Application
 ------------------------------------------------
 
 In this step, you'll use the {+driver-short+} 
-to connect to your MongoDB cluster and run a query on the sample data.
+to connect to your MongoDB cluster and run a query on the sample data. You'll need your
+preferred text editor or :wikipedia:`integrated development environment (IDE) <Integrated development environment>`
```

Comment:
S: we usually use a permanent link to Wikipedia pages!

```suggestion
preferred text editor or :wikipedia:`integrated development environment (IDE) <w/index.php?title=Integrated_development_environment&oldid=1220874232>`
```

Reply:
Addressed in new commit!